# debug object map

# ============================
# template
#
# start_time: start_time(sec)
# end_time: end_time(sec)
# objects:
#     - id: object_id
#       name: object_name
#       x: x_position
#       y: y_position
# ============================

debug_object_map:
    - start_time: 0
      end_time: 600
      objects:
          - id: 0
            name: trash_can
            x: 0.13
            y: -2.39
          - id: 1
            name: trash_can
            x: 0.11
            y: -1.97
          - id: 2
            name: trash_can
            x: 0.09
            y: -1.55
          - id: 3
            name: trash_can
            x: 0.04
            y: -1.18
          - id: 4
            name: trash_can
            x: 18.48
            y: -4.28
          - id: 5
            name: trash_can
            x: 18.45
            y: -3.84
          - id: 6
            name: trash_can
            x: 18.43
            y: -3.49
          - id: 7
            name: trash_can
            x: 18.41
            y: -3.04
          - id: 8
            name: trash_can
            x: -15.20
            y: -1.38
          - id: 9
            name: trash_can
            x: -15.56
            y: -1.42
          - id: 10
            name: trash_can
            x: -15.97
            y: -1.42
          - id: 11
            name: trash_can
            x: -16.33
            y: -1.44
          - id: 12
            name: trash_can
            x: -12.31
            y: -10.08
          - id: 13
            name: trash_can
            x: -12.29
            y: -10.42
          - id: 14
            name: trash_can
            x: -12.28
            y: -10.80
          - id: 15
            name: trash_can
            x: -12.23
            y: -11.19
          - id: 16
            name: fire_hydrant
            x: 11.57
            y: 1.20
          - id: 18
            name: fire_hydrant
            x: -6.63
            y: 0.58
          - id: 20
            name: fire_hydrant
            x: -5.53
            y: -15.27
          - id: 22
            name: fire_hydrant
            x: 12.60
            y: -14.62
          - id: 23
            name: fire_hydrant
            x: 24.50
            y: -14.25
    - start_time: 600
      end_time: 900
      objects:
          - id: 0
            name: trash_can
            x: 0.39
            y: -3.05
          - id: 1
            name: trash_can
            x: 0.38
            y: -2.64
          - id: 2
            name: trash_can
            x: 0.09
            y: -1.55
          - id: 3
            name: trash_can
            x: 0.41
            y: -3.47
          - id: 4
            name: trash_can
            x: 18.48
            y: -4.28
          - id: 5
            name: trash_can
            x: 18.45
            y: -3.84
          - id: 6
            name: trash_can
            x: 18.43
            y: -3.49
          - id: 7
            name: trash_can
            x: 18.41
            y: -3.04
          - id: 8
            name: trash_can
            x: -15.20
            y: -1.38
          - id: 9
            name: trash_can
            x: -15.56
            y: -1.42
          - id: 10
            name: trash_can
            x: -15.97
            y: -1.42
          - id: 11
            name: trash_can
            x: -16.33
            y: -1.44
          - id: 12
            name: trash_can
            x: -12.31
            y: -10.08
          - id: 13
            name: trash_can
            x: -12.29
            y: -10.42
          - id: 14
            name: trash_can
            x: -12.28
            y: -10.80
          - id: 15
            name: trash_can
            x: -12.23
            y: -11.19
          - id: 16
            name: fire_hydrant
            x: 11.57
            y: 1.20
          - id: 18
            name: fire_hydrant
            x: -6.63
            y: 0.58
          - id: 20
            name: fire_hydrant
            x: -5.53
            y: -15.27
          - id: 22
            name: fire_hydrant
            x: 12.60
            y: -14.62
          - id: 23
            name: fire_hydrant
            x: 24.50
            y: -14.25